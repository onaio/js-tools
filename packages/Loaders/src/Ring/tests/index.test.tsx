import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';
import React from 'react';

import { Ring } from '..';

describe('Loaders.Ring', () => {
  it('renders without crashing', () => {
    const wrapper = mount(<Ring />);

    expect(toJson(wrapper.find('Ring'))).toMatchSnapshot();
  });
});
