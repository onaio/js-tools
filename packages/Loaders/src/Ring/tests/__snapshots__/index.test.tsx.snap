// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loaders.Ring renders without crashing 1`] = `
.c0 {
  display: inline-block;
  position: relative;
  width: 80px;
  height: 80px;
}

.c0 div {
  box-sizing: border-box;
  display: block;
  position: absolute;
  width: 64px;
  height: 64px;
  margin: 8px;
  border: 8px solid tomato;
  border-radius: 50%;
  border-color: tomato transparent transparent transparent;
  -webkit-animation: cilQsd 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  animation: cilQsd 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
}

.c0 div:nth-child(1) {
  -webkit-animation-delay: -0.45s;
  animation-delay: -0.45s;
}

.c0 div:nth-child(2) {
  -webkit-animation-delay: -0.3s;
  animation-delay: -0.3s;
}

.c0 div:nth-child(3) {
  -webkit-animation-delay: -0.15s;
  animation-delay: -0.15s;
}

<Ring
  animationSpeed={2.4}
  color="tomato"
  scaleSizeBy={1}
  scaleSpeedBy={1}
>
  <styled.div
    animationSpeed={2.4}
    color="tomato"
    scaleSizeBy={1}
    scaleSpeedBy={1}
  >
    <div
      className="c0"
      color="tomato"
    >
      <div />
      <div />
      <div />
      <div />
    </div>
  </styled.div>
</Ring>
`;
